unblob/cli.py

```diff
@@ -60,10 +60,22 @@ def get_help_text():
 @click.option(
     "-d",
     "--depth",
-    type=int,
-    default=DEFAULT_DEPTH,
```

Comment:
I don't understand why we replaced the explicit constant definition by implicit `10` values being scattered around. Is there a reason behind it ?

Reply:
I removed it because the module did not use it where it was defined anymore, but you are right, I should have thought about this more :)